labels.yml

```diff
@@ -16,20 +16,40 @@ labels:
     color: 'f5f5f5'
 
   - name: monitoring
-    description: "This issue should be solved, but it should be monitored for a period before being closed"
+    description: "This issue should be solved, but let's keep an eye on it for a while before we close it."
     color: '357399'
 
   - name: performance
-    description:
+    description: "This issue significantly affects current performance, or will when we scale."
     color: '208116'
 
+  - name: operations
+    description: Anything related to our internal IT systems.
+    color: '262626'
+
+  - name: infrastructure
+    description: Product-related tooling.
+    color: '56a39f'
+
+  - name: debt
+    description: "When we originally built this feature, we knew we'd need to improve it later."
+    color: 'b51e1e'
+
   - name: refactor
-    description:
+    description: "New customer requirements, software updates, or changing business needs are motivating us to improve this."
     color: '627ada'
 
   - name: security
-    description:
-    color: '3b4047'
+    description: A significant customer-facing security issue.
```

Comment:
nitpicking on description - should it matter if its customer facing or not? any security issue has the potential to become customer facing. I'm also not sure if the word "significant" is necessary or not.